docs/content/docs/dev/table/sqlClient.md

```diff
@@ -812,4 +812,35 @@ Flink SQL> RESET pipeline.name;
 
 If the option `pipeline.name` is not specified, SQL Client will generate a default name for the submitted job, e.g. `insert-into_<sink_table_name>` for `INSERT INTO` statements.
 
+### Monitoring job status
```

Comment:
I think we can merge these two parts into one section like `Managing job lifecycle` and link to the specified page about this.

Reply:
I've also thought about it, but there's a minor issue if we do that: in the cli docs, `Managing job lifecycle` includes submitting jobs, which does not in SQL Client docs. WDYT? @fsk119

Reply:
The first time I read it, I thought this part was a bit repetitive. One idea gives an example of how users stop the job. Users first use SHOW statements to get the running job list and then select one of these to kill. How about adding a new section named `Stop a Job with a savepoint`?